data/system/assets/tools/uphill.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="34" height="29" viewBox="0 0 34 29" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I made this in Inkscape. Any better ideas for representing uphill? I thought about some kind of sweat/exhaustion animation, but no concrete ideas